kits/arm/ex_AR_kit.cpp

```diff
@@ -0,0 +1,223 @@
+/*
+ * AR Kit Example
+ * Currently assumes 6 degrees of freedom in you arm!
+ * This demo sets up your arm's end effector to mimick the changes in 
+ * position and orientation of your mobile IO device.
+ */
+
+#include "group_command.hpp"
+#include "group_feedback.hpp"
+#include "robot_model.hpp"
+#include "arm/arm.hpp"
+#include "util/mobile_io.hpp"
+#include <chrono>
+#include <thread>
+#include "hebi_util.hpp"
+
+using namespace hebi;
+using namespace experimental; // For all things mobileIO 
+
+
+
+Eigen::Matrix3d makeRotationMatrix (hebi::Quaternionf phone_orientation) {
+  Eigen::Quaterniond q;
+  q.w() = phone_orientation.getW();
+  q.x() = phone_orientation.getX();
+  q.y() = phone_orientation.getY();
+  q.z() = phone_orientation.getZ();
+  return q.toRotationMatrix();
+}
+
+
+int main(int argc, char* argv[])
+{
+  //////////////////////////
+  ///// Config Setup ///////
+  //////////////////////////
+
+  // Config file path
+  const std::string example_config_file = "config/ex_AR_kit.cfg.yaml";
+  std::vector<std::string> errors;
+  
+  // Load the config
+  const auto example_config = RobotConfig::loadConfig(example_config_file, errors);
+  for (const auto& error : errors) {
+    std::cerr << error << std::endl;
+  }
+  if (!example_config) {
+    std::cerr << "Failed to load configuration from: " << example_config_file << std::endl;
+    return -1;
+  }
+
+  // For this demo, we need the arm and mobile_io
+  std::unique_ptr<hebi::experimental::arm::Arm> arm;
```

Comment:
I think dropping the "experimental" here will mean fewer changes when we remove that namespace (hopefully soon!)

Reply:
Fixed!